specification/search/data-plane/Azure.Search/preview/2021-04-30-Preview/searchservice.json

```diff
@@ -6161,7 +6161,7 @@
     },
     "SearchIndexerDataNoneIdentity": {
       "description": "Clears the identity property of a datasource.",
-      "x-ms-discriminator-value": "#Microsoft.Azure.Search.SearchIndexerDataNoneIdentity",
+      "x-ms-discriminator-value": "#Microsoft.Azure.Search.DataNoneIdentity",
```

Comment:
That was a bug (https://github.com/Azure/azure-sdk-for-net/issues/29813), there is no such thing as `Microsoft.Azure.Search.SearchIndexerDataNoneIdentity`, so the back-end could not resolve that type and properly map the data, and consequently the API could not work with user-assigned managed identity. This can't break anything since no working code could exercise the previous value.

This is a preview version, that is available to the portal and to customers.

Reply:
Thanks. I approved the PR.

But still @JeffreyRichter on the bug fix in shape of breaking changes, in case we have objection.

Reply:
If the swagger was wrong, then we have to fix it, so I'd approve it.
But I don't see the BreakingChange label applied to this PR.

Reply:
Added an issue for this case: https://github.com/Azure/openapi-diff/issues/241